includes/Admin/Orders.php

```diff
@@ -343,18 +347,15 @@ private function get_cancel_modal_buttons() {
 	 * @internal
 	 *
 	 * @since 2.1.0-dev.1
-	 *
-	 * @param boolean $select_id id for the select HTML element
-	 * @param boolean $hidden whether or not the field should be hidden
 	 */
-	public function render_refund_reason_field( $select_id = 'wc_facebook_refund_reason', $hidden = true ) {
+	public function render_refund_reason_field() {
 
 		if ( ! $this->is_edit_order_screen() ) {
 			return;
 		}
 
 		?>
-		<select id="<?php echo esc_attr( $select_id ); ?>" <?php echo $hidden ? 'style="display: none;"' : ''; ?>>
```

Comment:
I remember to review a PR from Deiva in these lines. Were they intentionally changed? I'm asking because this handles the refund modal and not the complete order modal.

Reply:
Conflict resolved in 5927703dbae31f24e07d934986af7f999ea92507